### PR TITLE
add doppel-cli recipe

### PR DESCRIPTION
**Pull request by jameslamb**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jameslamb** [comment]
@conda-forge-admin, please ping team

File: recipes/doppel-cli/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@echo on
```
Comment (xhochy) [inline recipes/doppel-cli/bld.bat:1]: Doesn't using `pip` not work here?

File: recipes/doppel-cli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - click
+    - pip
+    - python
+    - tabulate
+  run:
+    - click
+    - python
+    - tabulate
+test:
+  requires:
```
Comment (xhochy) [inline recipes/doppel-cli/meta.yaml:26]: `pytest` is not required for the commands? Please remove.

File: recipes/doppel-cli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+{% set version = '0.3.1' %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+source:
+  url: "https://pypi.io/packages/source/d/doppel-cli/doppel-cli-{{ version }}.tar.gz"
+  sha256: ea363bba25f6f38f64e5a3a9d52d11e01483c1c532eb6bcd85eabbc89118cd1a
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/doppel-cli/meta.yaml:11]: Seems like this can be `noarch: python`.

**Comment by xhochy** [comment]
Might be helpful to generate this recipe using https://conda-forge.org/blog//2020/03/05/grayskull

**Comment by jameslamb** [comment]
> Might be helpful to generate this recipe using https://conda-forge.org/blog//2020/03/05/grayskull

hey thanks @xhochy ! I wish I knew about `grayskull` before, it's really great. Would have save me a lot of time.

On https://github.com/conda-forge/staged-recipes/commit/ae6dd9021ba5d273dbf3bb7bdfb55e337715abde I checked in the meta file generated by `grayskull pypi doppel-cli`. Seems like it addresses all of your comments

**Pull request merged**